HerPublicWebsite/appsettings.DEV.json

```diff
@@ -1,4 +1,8 @@
 {
+    "Global": {

```

Comment:
@Joe-Edwards As I understand, these will overwrite the values in `appsettings.json` if there are values provided in an enviroment specific config file, so this format will work? (so in this case the value in appsettings will only be used for local dev where it is not overwritten?)

Let me know if I've missed something.

Reply:
Yep exactly - `appsettings.json` is effectively the 'default' config, and the environment-specific ones will override it.

We should check it's working in a lower environment of course, but looks good to me :)

(Note `appsettings.DEV.json` is for the deployed dev environment, and `appsettings.development.json` is for local dev)